runtime/etc/logrotate.d/etcd

```diff
@@ -0,0 +1,11 @@
+/var/log/etcd.log
+{
+        rotate 1
```

Comment:
@mrandi Think about other people which may rely on having these logs.
My suggestion is at least have files for one week.